components/hab/static/template_init

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh
```

Comment:
From what I have seen, the vast majority of plans don't have hooks. While a commented hooks file is nice for seeing a working example, it just creates more work if the author is going to have to delete them in most cases.